components/Service/ServicePage.js

```diff
@@ -0,0 +1,172 @@
+                        <div className="intro-content">
+                            <i className="fas fa-first-aid icons-status" />
+                            <hr />
+                            <span className="services-content-text context4">
+                                <p className="intro-text1">
+                                   Quick First Aid Solutions
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                    </div>
+                    <div className="content-wrapper services-wrapper intro-wrap">
+                        <div className="intro-content">
+                            <i className="fas fa-hospital icons-status hos" />
+                            <hr />
+                            <span className="services-content-text context2">
+                                <p className="intro-text1">
+                                   List of Hospitals Closest to You
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                        <div className="intro-content">
+                            <i className="fas fa-tv icons-status" />
+                            <hr />
+                            <span className="services-content-text context1">
+                                <p className="intro-text1">
+                                Medicial Screenings
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                        <div className="intro-content">
+                            <i className="fas fa-ambulance icons-status" />
+                            <hr />
+                            <span className="services-content-text context1">
+                                <p className="intro-text1">
+                                   Emergency Ambulance
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div className="services app-content">
+                <div className="content-wrapper services-wrapper service-wrap">
+                    <div className="services-content">
+                        <img className="serviceI" src="/static/img/help.jpg" alt="Smiley face" />
+                        <span className="services-content-text">
+                            <p className="service-text1">
+                                Emergency Contact
+                                <br />
+                                {' '}
+                                Number
```

Comment:
What is this `Number` doing here?

Reply:
I don't understand your question

Reply:
You have `Number` on line 30, what is its reason for being there?